dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/WebModuleOpenAPIProvider.java

```diff
@@ -1,18 +1,13 @@
 package io.openliberty.microprofile.openapi20;
```

Comment:
copyright header

Reply:
This was missing the copyright header before. It looks like Martin created it in 2020, so I've put that as the initial date in the header.